src/main/java/uk/gov/hmcts/reform/em/orchestrator/automatedbundling/configuration/BundleConfiguration.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.hmcts.reform.em.orchestrator.automatedbundling.configuration;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+
+/**
+ * Parsed configuration file for automated bundles.
+ */
+public class BundleConfiguration {
+
+    public final String title;
+    public final List<BundleConfigurationFolder> folders;
+
+    public BundleConfiguration(@JsonProperty("title") String title,
```

Comment:
what's this doing?

Reply:
nvm, I'll ctrl click through it

Reply:
just tells jackson to assign the field "title" to this variable, otherwise it can't work it out